procrustes/psdp.py

```diff
@@ -23,18 +23,249 @@
 """Positive semidefinite Procrustes Module."""
 
 from math import inf, sqrt
-from typing import Optional
+from typing import Dict, Optional
 
 import numpy as np
 from numpy.linalg import multi_dot
 from procrustes.utils import compute_error, ProcrustesResult, setup_input_arrays
 import scipy
 from scipy.optimize import minimize
 
-__all__ = [
-    "psdp_woodgate",
-    "psdp_peng"
-]
+__all__ = ["psdp_woodgate", "psdp_peng", "psdp_opt"]
```

Comment:
I would put

```python
__all__ = [
    "psdp_woodgate",
    "psdp_peng",
    "psdp_opt",
]
```
This would help future new changes clean.

Reply:
Oh yeah! My autoformatter unfortunately updates this into a single line and I always forget to re-edit it. I will fix this right away.